Models/news.models.js

```diff
@@ -21,13 +21,18 @@ module.exports.fetchUsers = () => {
 
 module.exports.fetchArticlesByID = (article_id) => {
   return db
-    .query("SELECT * FROM articles WHERE article_id = $1", [article_id])
-    .then((result) => {
-      if (result.rows.length === 0) {
+    .query(
+      `SELECT *, CAST((SELECT COUNT(*) FROM comments WHERE comments.article_id = articles.article_id) AS INTEGER) AS comment_count
+      FROM articles 
+      WHERE article_id = $1`,
```

Comment:
Looks good. It's quite complicated though, using a sub-query. You could do this just with the `COUNT` aggregate function and the keyword `AS`, no need refactor